packages/zoe/test/swingsetTests/zoe/vat-bob.js

```diff
@@ -17,12 +16,15 @@ const build = async (log, zoe, issuers, payments, installations, timer) => {
   const [moolaPayment, simoleanPayment] = payments;
   const [moolaIssuer, simoleanIssuer, bucksIssuer] = issuers;
   const inviteIssuer = await E(zoe).getInviteIssuer();
-  const getInstanceHandle = makeGetInstanceHandle(inviteIssuer);
 
   return harden({
     doAutomaticRefund: async inviteP => {
       const invite = await inviteP;
       const exclInvite = await E(inviteIssuer).claim(invite);
+      const getInstanceHandle = iP =>
+        E(inviteIssuer)
+          .getAmountOf(iP)
+          .then(amount => amount.value[0].instanceHandle);
```

Comment:
This can use `zoeTestHelper.getInviteFields()` rather than being defined in many  files

Reply:
This hasn't been updated for the new design at all